### PR TITLE
<RadioButton> - Add unidriver

### PR DESCRIPTION
**Pull request by liorcode**

### CODE REVIEW

File: src/RadioGroup/RadioButton/RadioButton.uni.driver.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export const radioButtonUniDriverFactory = base => {
+  const reactBase = ReactBase(base);
+
+  const getRadioButton = async () => (await reactBase.children())[0];
+  const getLabel = async () => (await reactBase.children())[1];
+  const getContent = () => base.$('[data-hook="radio-button-content"]');
+
+  return {
+    ...baseUniDriverFactory(base),
+    check: async () => ReactBase(await getRadioButton()).change(),
```
Comment (mosheKerbel) [inline src/RadioGroup/RadioButton/RadioButton.uni.driver.js]: it means that consumers won't be able to use "check function in platforms other than React/jsdom.
Let's try to remove it and to use some combination of click and press the enter key and see if it works.

File: src/RadioGroup/RadioButton/RadioButton.uni.driver.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { baseUniDriverFactory, ReactBase } from '../../../test/utils/unidriver';
+
+export const radioButtonUniDriverFactory = base => {
+  const reactBase = ReactBase(base);
+
+  const getRadioButton = async () => (await reactBase.children())[0];
```
Comment (mosheKerbel) [inline src/RadioGroup/RadioButton/RadioButton.uni.driver.js]: try to replace it with the data-hook to radioButton

File: src/RadioGroup/RadioButton/RadioButton.uni.driver.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  const getLabel = async () => (await reactBase.children())[1];
```
Comment (mosheKerbel) [inline src/RadioGroup/RadioButton/RadioButton.uni.driver.js]: try to use the label data-hook

**Comment by mosheKerbel** [comment]
@liorcode plz resolve conflicts and then we can merge

**Comment by mosheKerbel** [comment]
@liorcode 
one of the async tests is  failing

image

**Comment by liorcode** [comment]
> @liorcode
> one of the async tests is failing
> 
> image

Weird, I tried running it several times and couldn't reproduce on my machine...is there anything special about the env it is running on that you can think of?

**Comment by mosheKerbel** [comment]
indeed wierd
did not happen on my machine also
will try to rebuild it on teamcity

**Comment by mosheKerbel** [comment]
note to self - now reproduces on local machine
seems like click() of UniDriver is not triggering the onChange event of a radio button
need to figure it out.

**Pull request closed without merging**